src/dapps/catalog.ts

```diff
@@ -0,0 +1,284 @@
+import {
+  generateLazyValidator,
+  JSONSchema,
+  ValidateFunction
+} from '../validation'
+import { ChainId } from './chain-id'
+import { BodyShape, EmoteCategory } from '../platform'
+import { Network } from './network'
+import { NFTCategory } from './nft-category'
+import { Rarity } from './rarity'
+import { WearableCategory } from './wearable-category'
+import { Item, ItemFilters } from './item'
+
+export type CatalogItem = Pick<
+  Item,
+  | 'id'
+  | 'name'
+  | 'contractAddress'
+  | 'thumbnail'
+  | 'url'
+  | 'rarity'
+  | 'category'
+  | 'creator'
+  | 'data'
+  | 'network'
+  | 'chainId'
+  | 'available'
+  | 'isOnSale'
+  | 'price'
+  | 'picks'
```

Comment:
Thanks for this! Would you mind adding the structure to the schemas?

Reply:
done! :)